solutions/hw5/treap.py

```diff
@@ -0,0 +1,122 @@
+from collections import MutableMapping
+from typing import Iterator, Callable, Any
+from random import randint
+
+
+class Node:
```

Comment:
Sorry for my nerdy behaviour, but I would like to see docstrings in the code at least for key structures - Node, Deramida